src/metabase/collections/api.clj

```diff
@@ -1418,3 +1346,80 @@
                  (decode-graph {:revision revision :groups groups})
                  skip-graph
                  force))
+
+;;; ------------------------------------------ Fetching a single Collection -------------------------------------------
+
+(api.macros/defendpoint :get "/:id"
+  "Fetch a specific Collection with standard details added"
+  [{:keys [id]} :- [:map
+                    [:id [:or ms/PositiveInt ms/NanoIdString]]]]
+  (let [resolved-id (eid-translation/->id-or-404 :collection id)]
+    (collection-detail (api/read-check :model/Collection resolved-id))))
+
+(api.macros/defendpoint :put "/:id"
```

Comment:
Moved here from above to be consistent with `GET /collection/:id` above. Changed nothing.